TabloidMVC/Views/Tags/Index.cshtml

```diff
@@ -0,0 +1,41 @@
+﻿@model IEnumerable<TabloidMVC.Models.Tags>
+
+@{
+    ViewData["Title"] = "Index";
+}
+
+<h1>Index</h1>
+
+<p>
+    <a asp-action="Create">Create New</a>
+</p>
+<table class="table">
+    <thead>
+        <tr>
+            <th>
+                @Html.DisplayNameFor(model => model.Id)
+            </th>
+            <th>
+                @Html.DisplayNameFor(model => model.Name)
+            </th>
+            <th></th>
+        </tr>
+    </thead>
+    <tbody>
+@foreach (var item in Model) {
+        <tr>
+            <td>
+                @Html.DisplayFor(modelItem => item.Id)
+            </td>
+            <td>
+                @Html.DisplayFor(modelItem => item.Name)
+            </td>
+            <td>
+                @Html.ActionLink("Edit", "Edit", new { /* id=item.PrimaryKey */ }) |
```

Comment:
Right now, these buttons throw exceptions when you click on them. I know you'll get to them on a later ticket, but I want y'all to imagine that merging to master is essentially handing the app over to the client-- as soon as you merge, the end user can start clicking on things. So right now, if we merged, the end user would be able to click these buttons and break stuff. 

Let's remove these buttons until we get to their tickets.

Reply:
@mattspence338 Take these buttons off! They break stuff! ☝️